### PR TITLE
[FIX] web: reload search panel before view model

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/js/views/abstract_controller.js [hunk 1]
```diff
@@ -211,7 +211,6 @@ var AbstractController = mvc.Controller.extend(ActionMixin, {
      * @returns {Promise}
      */
```
Comment (Polymorphe57) [inline addons/web/static/src/js/views/abstract_controller.js:212]: to the Problem

File: addons/web/static/tests/views/search_panel_tests.js [hunk 2]
```diff
@@ -3993,5 +3994,77 @@ QUnit.module('Views', {
 
         kanban.destroy();
     });
+
+    QUnit.test("wait search panel to be reloaded before model reload", async function (assert) {
```
Comment (Polymorphe57) [inline addons/web/static/tests/views/search_panel_tests.js:3998]: bad title?

File: addons/web/static/src/js/views/abstract_controller.js (hunk 1 above)
Comment (Polymorphe57) [reply addons/web/static/src/js/views/abstract_controller.js:212]: done

File: addons/web/static/tests/views/search_panel_tests.js (hunk 2 above)
Comment (Polymorphe57) [reply addons/web/static/tests/views/search_panel_tests.js:3998]: done

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**